Lib/uuid.py

```diff
@@ -455,8 +458,18 @@ def _netstat_getnode():
                 try:
                     words = line.rstrip().split()
                     word = words[i]
-                    if len(word) == 17 and word.count(b':') == 5:
-                        mac = int(word.replace(b':', b''), 16)
+                    if word.count(_MAC_DELIM) == 5:
+                        if len(word) == 17:
+                            mac = int(word.replace(_MAC_DELIM, b''), 16)
+                        # the extracted hex string will not be a 12 hex digit
+                        # string, so extract the fields and add them in
+                        # piece by piece
+                        elif len(word) < 17 and len(word) >= 11:
```

Comment:
Please revert this change.

If you want to keep it, please make it specific to AIX and write an unit test for it.

Reply:
made specific for AIX